### ISSUE TITLE
Current versions for OpenAI `gym` and other packages -- `PongNoFrameskip-v4` is not found in current `gym`, `atari-py` 

### ISSUE THREAD

**Issue by 00krishna** [PacktPublishing/Deep-Reinforcement-Learning-Hands-On#100]
Hello Folks. 

I was just wondering how to adapt the repo code to the current versions of `gym` and `atari-py`. It seems that a lot of the classic 
RL control environments were migrated to the `ALE` packages and some other things. I tried to use the `PongNoFrameskip-v4` environment, but I kept getting an error message about the environment not being registered or indexed--even though I did install the ROMs before, etc. I was testing out the new `Pong-v5` or something environment to see if that will work instead, but still just trying to get it to work. 

Any suggestions would be appreciated. Of course trying to keep up with all of the versions is a full time job, right. But it seems like so many of the other tutorials out there are based upon the older version of OpenAI as well, so no luck finding other implementations to emulate. 

Thanks.

**Comment by dayemsiddiqui** [comment]
Hi I am facing the same issue. Did you manage to find some solution?

**Comment by 00krishna** [comment]
@dayemsiddiqui Yeah, I believe that this environment has been moved to a new package called `ale-py`. I have not tried it myself, as I was using the `Pong-v5` or something version from `gym`. But someone told me that these legacy environments were moved to the new package. I know the documentation was not really updated anywhere.
